mappings/net/minecraft/block/material/Material.mapping

```diff
@@ -0,0 +1,61 @@
+CLASS none/ayo net/minecraft/block/material/Material
+	CLASS none/ayo$1
+		METHOD c isSolid ()Z
+	FIELD A CACTUS Lnone/ayo;
+	FIELD B OTHER Lnone/ayo;
+	FIELD C PLANT_BLOCK Lnone/ayo;
```

Comment:
It does in¢lude flowers. The main difference between `PLANT` and `PLANT_FLAMMABLE` is the flammability. The main difference between those and `PLANT_BLOCK` is that the latter uses full cubes.